loghub/templates/release.txt

```diff
@@ -0,0 +1,21 @@
+## Version {{ version }} ({{ close_date }})
+
+{%- if issues|length %}
+### Issues Closed
+
+{%-   for i in issues %}
+* Issue #{{ i['number'] }} - {{ i['title'] }}
+{%-   endfor %}
+
+In this release {{ issues|length }} issue{% if issues|length != 1 %}s{% endif %} {% if pull_requests|length == 1 %}was{% else %}were{% endif %} closed.
+{%- endif %}
+                
+{%- if pull_requests|length %}
+### Pull Requests Merged
+
+{%-   for pr in pull_requests %}
+* PR #{{ pr['number'] }} - {{ pr['title'] }}
+{%-   endfor %}
+
+In this release {{ pull_requests|length }} pull request{% if pull_requests|length != 1 %}s{% endif %} {% if pull_requests|length == 1 %}was{% else %}were{% endif %} closed.
+{%- endif %}
```

Comment:
This doesn't take into account the logic I left in place to avoid printing sections that don't have any content at all